### ISSUE TITLE
tracker URL

### ISSUE THREAD

**Issue by ariesjj** [SuprDewd/simpletracker#17]
great work bro...what is the tracker url?

**Comment by Wiilf** [comment]
Yeah what is the URL. Everything is fine on user side except no clarity on this step when creating a torrent.

**Comment by Wiilf** [comment]
Yeah what is the URL. Everything is fine on user side except no clarity on this step when creating a torrent.

**Comment by Wiilf** [comment]
Yeah what is the URL. Everything is fine on user side except no clarity on this step when creating a torrent.

**Comment by Wiilf** [comment]
Yeah what is the URL. Everything is fine on user side except no clarity on this step when creating a torrent.

**Comment by Wiilf** [comment]
Yeah what is the URL. Everything is fine on user side except no clarity on this step when creating a torrent.

**Comment by SuprDewd** [comment]
Sorry, maybe this should be made more clear in the upload interface. But in reality it doesn't matter at all what tracker/announce URL you use; it is automatically populated in [download.php](https://github.com/SuprDewd/simpletracker/blob/master/serve/download.php#L34) when other users download your torrent file.

**Comment by Wiilf** [comment]
It’s okay, but I’m moving 23,000 to this tracker so I’m hours at the changes. From what I gather, announce is relying on get parameters (port, ip, etc all client default) but the restrictions come in with torrent_id and username, despite falsifying values in the key array. 

So I attempt to just read passkey from user table and adjust accordingly, HTTP Error. 

It would be nice to contribute, I’ve mad mass uploader that stores all torrents in dir to blob, many other changes in signup and index. I don’t know how git hub works.

**Comment by Wiilf** [comment]
If you don't mind answering this last question, when creating torrent what announce URL format should be used? The initial seeder client must have communication with the peer table. This is the confusion. 

Once uploaded to the site, everything works as expected but downloaded .torrents from download.php never connect to the seeder client, therefore peer details for both Seeder and Downloader is never inserted (only downloader). I had both appear only once, but I had to trial many different announce params in guessing. What do you do personally in this situation? Do you reply on IP:Port etc?

I appreciate your response and time.

**Comment by Wiilf** [comment]
Yes all is well. I have figured it out. For those curious, download will reinsert a new announce URL, whether created torrent has announce or is blank. From there username and torrent_id is matched to peer table and the rest is sorted.

I'll be modifying this so it retrieves only passkey, allowing JOINs to retrieve other fields away from _GET in client. Thanks again! This is a great script and I plugged it in very nicely. 

Working with user dB session class, mass-uploader and more. Thank you for no UI, it made back-porting extremely easy.

**Comment by SuprDewd** [comment]
Hey, do you mind dropping me an email about your plans at suprdewd@gmail.com
?
